.rubocop_todo.yml

```diff
@@ -1,56 +1,7 @@
-Metrics/CyclomaticComplexity:
+Metrics/BlockLength:
   Exclude:
-    - 'lib/sufia/arkivo/metadata_munger.rb'
-
-Metrics/ClassLength:
-  Exclude:
-    - 'lib/generators/sufia/templates/catalog_controller.rb'
-    - 'lib/generators/sufia/install_generator.rb'
-    - 'app/services/sufia/user_stat_importer.rb'
-
-Metrics/ParameterLists:
-  Exclude:
-    - 'app/jobs/batch_create_job.rb'
-
-Metrics/ModuleLength:
-  Exclude:
-    - 'app/controllers/concerns/sufia/users_controller_behavior.rb'
-    - 'app/helpers/sufia/sufia_helper_behavior.rb'
-
-Metrics/MethodLength:
-  Exclude:
-    - 'lib/generators/sufia/templates/migrations/create_local_authorities.rb'
-    - 'app/helpers/sufia/citations_behaviors/formatters/endnote_formatter.rb'
-    - 'app/models/concerns/sufia/solr_document/export.rb'
-    - 'app/models/local_authority.rb'
-    - 'lib/generators/sufia/install_generator.rb'
-    - 'lib/sufia/arkivo/metadata_munger.rb'
-
-Style/HashSyntax:
-  Exclude:
-    - 'lib/generators/sufia/templates/catalog_controller.rb'
-
-Style/LineEndConcatenation:
-  Exclude:
-    - 'spec/test_app_templates/lib/generators/**/*'
-    - 'lib/generators/**/*'
-
-Style/PredicateName:
-  Exclude:
-    - 'app/helpers/sufia/sufia_helper_behavior.rb'
-    - 'app/controllers/concerns/sufia/controller.rb'
-
-Rails/Output:
-  Exclude:
-    - 'lib/generators/**/*'
-    - 'app/services/sufia/user_stat_importer.rb'
-
-Rails/HasAndBelongsToMany:
-  Exclude:
-    - 'app/models/domain_term.rb'
-    - 'app/models/local_authority.rb'
-
-Lint/HandleExceptions:
-  Exclude:
-    - 'lib/sufia/engine.rb'
+    - 'config/initializers/simple_form_bootstrap.rb'
```

Comment:
These two files are way way over the block length limit. Just disable them entirely.Maybe we should revisit this?